requirements.txt

```diff
@@ -0,0 +1 @@
+libevdev
```

Comment:
@SelfRef I would prefer to have a specific `python3` instance (virtual environment) for this package with a defined version of `libevdev`

Reply:
I added it with vent in mind but maybe it should be out of scope of this PR.

Deleted.